astroquery/alma/core.py

```diff
@@ -630,6 +629,20 @@ def get_data_info(self, uids, *, expand_tarfiles=False,
 
         return result
 
+    def is_datalink_adhoc_service(self, adhoc_service):
+        standard_id = self.get_adhoc_service_parameter(adhoc_service, 'standardID')
+        return standard_id == DATALINK_STANDARD_ID
+
+    def get_adhoc_service_access_url(self, adhoc_service):
+        return self.get_adhoc_service_parameter(adhoc_service, 'accessURL')
+
+    def get_adhoc_service_parameter(self, adhoc_service, parameter_id):
+        for p in adhoc_service.params:
+            if p.ID == parameter_id:
+                return p.value
+
+        return None
```

Comment:
A trailing `return` or `return None` is redundant
```suggestion
```

Reply:
[PEP 8](https://peps.python.org/pep-0008/) says:
> ... If any return statement returns an expression, any return statements where no value is returned should explicitly state this as `return None`, and an explicit return statement should be present at the end of the function (if reachable)...

Reply:
Huh.  Thanks.  This is the first time I think I really disagree with pep8, but ok.

Reply:
pep is only a recommendation not scripture, so on this one I agree with @keflavich, please remove the `return None`

Reply:
Reading a bit up on it, yes, maybe we need to add these back for all over the library, but atm pycodestyle/flake8 is not complaining, and if we run and check something more strict like pylint, we'll suffocating under linter errors.

So I removed the line (after all the only aim for this particular PEP8 rule is to be consistency, and removing this line is in fact more consistent with the rest of the library), and will merge once CI pass.